### PR TITLE
Particle 10.8.0 :: TAILWIND

### PR DESCRIPTION
**Pull request by jryanconklin**
Particle 10.8.0

Remove Bootstrap and Sass in favor of Tailwind.

### CODE REVIEW

**Comment by jryanconklin** [comment]
> @jryanconklin PR looks good! Just one thing, make sure ton run `npm run fmt` as there are a number of linting errors

Are you sure? I ran it and not seeing anything. I did inherit this branch and it had some spacing / line break weirdness going on. Think I fixed most of it.

File: webpack.particle.js [hunk 1]
```diff
@@ -7,10 +7,9 @@
 
 // Library Imports
 const { ProgressPlugin, ProvidePlugin } = require('webpack');
-const sass = require('sass');
 
 // Plugins
-const StyleLintPlugin = require('stylelint-webpack-plugin');
+// const StyleLintPlugin = require('stylelint-webpack-plugin');
```
Comment (yuginaili) [inline webpack.particle.js:12]: is this meant to be deleted?

File: webpack.particle.js [hunk 2]
```diff
@@ -146,8 +134,8 @@ module.exports = {
       'window.jQuery': 'jquery',
       Popper: ['popper.js', 'default'],
     }),
-    // Yell at us while writing Sass
-    new StyleLintPlugin(),
+    // // Yell at us while writing Sass
+    // new StyleLintPlugin(),
```
Comment (yuginaili) [inline webpack.particle.js:138]: same here

File: source/default/_patterns/01-atoms/image/_image--picture.twig [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
-{% set bp = scssBreakpoints|keys %}
+{% set bp = cssBreakpoints|keys %}
 
 <picture>
-  {% for item in image.srcset %}
-    {% set key = bp[loop.index]|default('xl') %}
-    <source srcset="{{ item }}" media="(max-width: {{ scssBreakpoints[key] }})">
-  {% endfor %}
-  {% include '@atoms/image/_image--primary.twig' with {
+	{% for item in image.srcset %}
```
Comment (illepic) [inline source/default/_patterns/01-atoms/image/_image--picture.twig:17]: Indentation here.

File: source/default/_patterns/01-atoms/svg/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# SVG Pattern
```
Comment (illepic) [inline source/default/_patterns/01-atoms/svg/README.md:1]: 💥 💥 💥 💥 💥 💥 💥 💥

File: source/default/webpack.config.js [hunk 5]
```diff
@@ -2,51 +2,46 @@
...
+                  tailwindConfig: path.resolve(__dirname, 'tailwind.config.js'),
+                  // PostCSS PurgeCSS config for Tailwind
+                  purgeCssConfig: {
+                    content: [
+                      path.resolve(__dirname, '_meta/**/*.*'),
+                      path.resolve(__dirname, '_patterns/**/*.*'),
+                    ],
+                    defaultExtractor: (content) =>
+                      content.match(/[A-Za-z0-9-_:/]+/g) || [],
+                    extensions: ['yml', 'twig', 'jsx'],
```
Comment (illepic) [inline source/default/webpack.config.js:29]: Maybe `json` here?

File: tools/generators/new-component/generators/app/templates/ds/index.js.ejs [hunk 6]
```diff
@@ -1,56 +1,19 @@
...
-  const settings = {
-    ...defaults,
-    ...<%= camelCaseName %>,
-  };
-  // An example of what could be done with this component
-  $<%= camelCaseName %>.addClass(settings.dummyClass);
-}
-
-export default enable;
+/** * <%= name %> */ import $ from 'jquery'; // Module dependencies import
```
Comment (illepic) [inline tools/generators/new-component/generators/app/templates/ds/index.js.ejs:1]: Uh oh.

File: apps/node-pl/pattern-lab/_patterns/00-protons-demo/borders.twig [hunk 7]
```diff
@@ -1,96 +1,145 @@
...
-      </div>
-      <hr>
-
-      <h2 class="py-3">Border</h2>
-      <p>Add classes to an element to remove all borders or some borders.</p>
-      <div class="border">
-        {% for class in border_classes %}
-          <span class=" bg-light m-2 border border{{ class }}"
-                style="display: inline-block; height: 5rem; width: 5rem; margin: .25rem;"></span>
+  {% block content %}
```
[the remark this answers is not in the document]
Comment (jryanconklin) [reply apps/node-pl/pattern-lab/_patterns/00-protons-demo/borders.twig:21]: Oh gosh, this is totally correct. The `content` block is below `usage` block. Blind on calls.

**Comment by bowiechang** [comment]
Trying to understand why was sass removed in this particular version? I believe sass is optional? And can be used with Tailwind. Would love any reply :)

**Pull request merged**